lottie/src/main/java/com/airbnb/lottie/animation/content/FillContent.java

```diff
@@ -120,6 +121,13 @@ public FillContent(final LottieDrawable lottieDrawable, BaseLayer layer, ShapeFi
       dropShadowAnimation.applyTo(paint);
     }
 
+    // Set layer paint blend mode
+    final BlendModeCompat blendMode = layer.getBlendMode().toNativeBlendMode();
+    if (blendMode != null) // PaintCompat.setBlendMode accepts null but no surprises please
+    {
+      PaintCompat.setBlendMode(paint, blendMode);
```

Comment:
Can the blend mode change? If not, can we do it once in the constructor instead of on every frame?